package.json

```diff
@@ -20,15 +20,15 @@
   },
   "homepage": "https://github.com/KyberNetwork/smart-contracts#readme",
   "dependencies": {
-    "@nomiclabs/buidler-truffle5": "1.0.2",
+    "@nomiclabs/buidler-truffle5": "^1.1.2",
```

Comment:
this is for error logs?
anyway
usually its better to commit web3 updates in separate commits.
and its better to do it for main branch.
since we will later do the same PR for development and Master....

so please don't add it here
rather do it in development branch.

also remove the ^
we want to know exactly which package we use.
And we want to be consistent across all runs.

Reply:
Yes, for error logging and also running buidler tests. This commit already removed from the commit tree.